api/geth_backend.go

```diff
@@ -2106,6 +2107,6 @@ func (b *GethStatusBackend) SwitchFleet(fleet string, conf *params.NodeConfig) e
 	return nil
 }
 
-func (b *GethStatusBackend) SetLocalPairing(value bool) {
-	b.localPairing = value
+func (b *GethStatusBackend) LPSM() *statecontrol.ProcessStateManager {
```

Comment:
Do you think replace `LPSM` to `LocalPairingStateManager` or `GetLocalPairingStateManager` looks better? LPSM looks too short for me ... 😥

Reply:
Ok, Just for you @qfrank I will replace this function with an exported field on the `GethStatusBackend` struct called `LocalPairingStateManager`

Reply:
This is done @qfrank